chunker/rdf_parser.go

```diff
@@ -226,6 +226,16 @@ L:
 		return rnq, errors.Errorf("NQuad failed sanity check:%+v", rnq)
 	}
 
+	// TODO(Naman): Ensure the label contains valid namespace.
+	// Append the namespace to the predicate before returning NQuad.
+	if rnq.Label != "" {
+		ns, err := strconv.ParseUint(rnq.Label, 0, 64)
+		if err != nil {
+			return rnq, err
+		}
+		rnq.Namespace = ns
```

Comment:
Do we need a separate field in NQuad just for the namespace? We can use the label field?
The label is `string` and Namespace is `uint64`.

Reply:
In the current implementation, the user cannot pass non-namespace strings in the label.
We can allow it in the following way:
The user tells that his RDFs contain a label and also tell us to load the data into a specific namespace. Then while parsing the RDFs we will not treat the label as namespace.